### PR TITLE
Convert Newtonsoft converters to STJ converters

### PR DESCRIPTION
**Pull request by hishamco**
Addresses part of #14456

### CODE REVIEW

File: src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentItemConverter.cs [hunk 1]
```diff
@@ -105,9 +77,64 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             return contentItem;
         }
 
-        public override bool CanConvert(Type objectType)
+        public override void Write(Utf8JsonWriter writer, ContentItem value, JsonSerializerOptions options)
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentItemConverter.cs:80]: This is might be a temp solution I'm working to add a `JTokenConverter`

**Comment by sebastienros** [comment]
The unit tests seems to fail because an entity has a `JObject`. This should become a `JsonDocument` (or `JsonElement`).

**Comment by sebastienros** [comment]
Unless here you are trying to keep the `JObject` and just use STJ to do the de/serialization which might be harder, though a totally valid approach.

**Comment by hishamco** [comment]
I might defer `ContentItemConverter` after we change `IEntity`

**Comment by MikeAlhayek** [comment]
@hishamco is this still needed? If not, please close it

**Comment by hishamco** [comment]
Nope, also there's another PR from @sarahelsaig need to be deleted too

**Comment by sarahelsaig** [comment]
Thanks for pointing it out, Hisham. I closed it.

**Pull request closed without merging**